nzedb/processing/tv/TMDB.php

```diff
@@ -0,0 +1,347 @@
+<?php
+namespace nzedb\processing\tv;
+
+use libs\Tmdb\TmdbAPI;
+use nzedb\ReleaseImage;
+
+class TMDB extends TV
+{
+	const MATCH_PROBABILITY = 75;
+
+
+	/**
+	 * @string DateTimeZone Object - UTC
+	 */
+	private $timeZone;
+
+	/**
+	 * @string MySQL DATETIME Format
+	 */
+	private $timeFormat;
+
+	/**
+	 * @var string The URL for the image for poster
+	 */
+	private $posterUrl;
+
+	/**
+	 * Construct. Instantiate TMDB Class
+	 *
+	 * @param array $options Class instances.
+	 *
+	 * @access public
+	 */
+	public function __construct(array $options = [])
+	{
+		parent::__construct($options);
+		$this->client = new TmdbAPI($this->pdo->getSetting('tmdbkey'));
+		$this->timeZone = new \DateTimeZone('UTC');
+		$this->timeFormat = 'Y-m-d H:i:s';
+	}
+
+	/**
+	 * Fetch banner from site.
+	 *
+	 * @param $videoId
+	 * @param $siteID
+	 *
+	 * @return bool
+	 */
+	public function getBanner($videoId, $siteID)
+	{
+		return false;
+	}
+
+	/**
+	 * Main processing director function for TMDB
+	 * Calls work query function and initiates processing
+	 *
+	 * @param            $groupID
+	 * @param            $guidChar
+	 * @param            $processTV
+	 * @param bool|false $local
```

Comment:
For getBanner?  GH doesn't really tell me what you're talking about specifically.